packages/itwinui-css/backstop/tests/footer.html

```diff
@@ -41,54 +41,54 @@ <h1>Footer</h1>
     <div id="demo-content"></div>
 
     <footer class="iui-legal-footer">
-      <ul>
-        <li>
+      <ul class="iui-legal-footer-list">
+        <li class="iui-legal-footer-item">
           ©&nbsp;
           <script>
             document.write(new Date().getFullYear());
           </script>
           &nbsp;Bentley Systems, Incorporated
-          <span class="iui-separator"></span>
         </li>
-        <li>
+        <li class="iui-legal-footer-separator"></li>
```

Comment:
You might be able to remove this element & class and instead use a pseudo element.

Reply:
I was thinking about that but idk, I thought if someone wants to use custom element as footer item, then having separator is kind of useful.